.nucleus.yaml

```diff
@@ -3,10 +3,77 @@ core-deploy:
   project-modules:
     lwc: lwc.version
 branches:
+  release:
```

Comment:
You can avoid repeating yourself using YAML anchors and references. 😉

Reply:
I had no idea this was a thing. I'll look into it.

Reply:
We are already using anchor and references in `.circleci/config.yml` to avoid repeating some properties (even if we can do something better). The official YAML documentation is really sparse on this subject, here are a couple of interesting links that might help you:
- https://support.atlassian.com/bitbucket-cloud/docs/yaml-anchors/
- https://medium.com/@kinghuang/docker-compose-anchors-aliases-extensions-a1e4105d70bd

Reply:
Used anchors to remove repetition.